<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from cogimon.srdf.xacro             | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<!--This does not replace URDF, and is not an extension of URDF.
    This is a format for representing semantic information about the robot structure.
    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
-->
<robot name="cogimon" xmlns:xacro="http://ros.org/wiki/xacro">
  <group name="base">
    <link name="Waist"/>
  </group>
  <group name="hands">
    <joint name="r_wrist_joint"/>
    <joint name="l_wrist_joint"/>
  </group>
  <group name="force_torque_sensors">
    <joint name="l_arm_ft_joint"/>
    <joint name="r_arm_ft_joint"/>
    <joint name="l_leg_ft_joint"/>
    <joint name="r_leg_ft_joint"/>
  </group>
  <group name="imu_sensors">
    <link name="imu_link"/>
  </group>
  <group name="left_leg">
    <chain base_link="Waist" tip_link="LFoot"/>
  </group>
  <group name="right_leg">
    <chain base_link="Waist" tip_link="RFoot"/>
  </group>
  <group name="torso">
    <chain base_link="Waist" tip_link="DWYTorso"/>
  </group>
  <group name="left_arm">
    <chain base_link="DWYTorso" tip_link="LWrMot3"/>
  </group>
  <group name="right_arm">
    <chain base_link="DWYTorso" tip_link="RWrMot3"/>
  </group>
  <group name="chains">
    <group name="left_leg"/>
    <group name="right_leg"/>
    <group name="torso"/>
    <group name="left_arm"/>
    <group name="right_arm"/>
  </group>
  <group_state name="home" group="chains">
    <joint name="head_roll" value="0.0"/>
    <joint name="WaistYaw" value="0.0"/>
    <joint name="WaistLat" value="0.0"/>
    <joint name="RShSag" value="0.959931"/>
    <joint name="RShLat" value="-0.007266"/>
    <joint name="RShYaw" value="-0.0"/>
    <joint name="RElbj" value="-1.919862"/>
    <joint name="RForearmPlate" value="0.0"/>
    <joint name="RWrj1" value="-0.523599"/>
    <joint name="RWrj2" value="-0.0"/>
    <joint name="LShSag" value="0.959931"/>
    <joint name="LShLat" value="0.007266"/>
    <joint name="LShYaw" value="-0.0"/>
    <joint name="LElbj" value="-1.919862"/>
    <joint name="LForearmPlate" value="0.0"/>
    <joint name="LWrj1" value="-0.523599"/>
    <joint name="LWrj2" value="-0.0"/>
    <joint name="RHipLat" value="0.0"/>
    <joint name="RHipSag" value="-0.363826"/>
    <joint name="RHipYaw" value="0.0"/>
    <joint name="RKneePitch" value="0.731245"/>
    <joint name="RAnklePitch" value="-0.307420"/>
    <joint name="RAnkleRoll" value="-0.0"/>
    <joint name="LHipLat" value="-0.0"/>
    <joint name="LHipSag" value="-0.363826"/>
    <joint name="LHipYaw" value="0.0"/>
    <joint name="LKneePitch" value="0.731245"/>
    <joint name="LAnklePitch" value="-0.307420"/>
    <joint name="LAnkleRoll" value="0.0"/>
  </group_state>
</robot>

